### ISSUE TITLE
[Header] Top-attached headers missing top border

### ISSUE THREAD

**Issue by donaggio** [fomantic/Fomantic-UI#180]
# Bug Report 

Top-attached headers lose top-border when there is other content before them, due to this CSS rule:

```
.ui.top.attached.header:not(:first-child) {
    border-top: none;
}
```

## Testcase
https://jsfiddle.net/n5qb9srL/

## Version
2.6.2

**Comment by hammy2899** [comment]
This can also be replicated here https://fomantic-ui.com/elements/header.html#attached

**Comment by hammy2899** [comment]
Fixed in 2.6.3
